apps/site/components/Common/Searchbox/Search.tsx

```diff
@@ -0,0 +1,227 @@
+                          : `/${locale}/${hit.document.path}`
+                      }
+                    >
+                      <DocumentTextIcon />
+                      <div>
+                        {typeof hit.document?.pageSectionTitle === 'string' && (
+                          <h3 className={styles.searchResultsItemTitle}>
+                            {hit.document?.pageSectionTitle}
+                          </h3>
+                        )}
+                        {typeof hit.document?.pageSectionTitle === 'string' &&
+                          typeof hit.document?.path === 'string' && (
+                            <p className={styles.searchResultsItemDescription}>
+                              {getFormattedPath(
+                                hit.document?.path,
+                                hit.document?.pageSectionTitle
+                              )}
+                            </p>
+                          )}
+                      </div>
+                    </Link>
+                  </SearchResults.Item>
+                )}
+              </SearchResults.GroupList>
+            </div>
+          )}
+        </SearchResults.GroupsWrapper>
+      </div>
+
+      <div className={styles.footer}>
+        <div className={styles.shortcutWrapper}>
+          <div className={styles.shortcutItem}>
+            <kbd className={styles.shortcutKey}>
+              <ArrowTurnDownLeftIcon />
+            </kbd>
+            <span className={styles.shortcutLabel}>
+              {t('components.search.keyboardShortcuts.select')}
+            </span>
+          </div>
+          <div className={styles.shortcutItem}>
+            <kbd className={styles.shortcutKey}>
+              <ArrowDownIcon />
+            </kbd>
+            <kbd className={styles.shortcutKey}>
+              <ArrowUpIcon />
+            </kbd>
+            <span className={styles.shortcutLabel}>
+              {t('components.search.keyboardShortcuts.navigate')}
+            </span>
+          </div>
+          <div className={styles.shortcutItem}>
+            <kbd className={styles.shortcutKey}>esc</kbd>
+            <span className={styles.shortcutLabel}>
+              {t('components.search.keyboardShortcuts.close')}
+            </span>
+          </div>
+        </div>
+        <div>
+          <a
+            href="https://www.orama.com/?utm_source=nodejs.org&amp;utm_medium=powered-by"
+            target="_blank"
+            rel="noopener noreferrer"
+            className={styles.poweredByLink}
+            data-focus-on-arrow-nav
+          >
+            <small>{t('components.search.poweredBy')}</small>
+            <Image
+              src="/static/logos/orama-logo-icon.svg"
+              alt={t('components.search.poweredBy') + ' Orama'}
+              width="22"
+              height="15"
+            />
+            <Image
+              src="/static/logos/orama-logo.svg"
+              alt={t('components.search.poweredBy') + ' Orama'}
```

Comment:
```suggestion
              alt={t('components.search.poweredBy')}
```

Can we put "Orama" in the translation?